### PR TITLE
feat(notification): initial version of Notification Component

### PR DESCRIPTION
**Pull request by andrewleyva**
# problem statement

need Notification Component

# solution
build it with Incedo.

This is WIP - do not merge

### CODE REVIEW

**Comment by wa11-e** [comment]
Woohoo! new surge deployment available for viewing! :tada: [surgereviewtzuywofgsztfhom.surge.sh](http://surgereviewtzuywofgsztfhom.surge.sh)

**Comment by wa11-e** [comment]
Woohoo! new surge deployment available for viewing! :tada: [surgereviewojtjrlbofrfvyfp.surge.sh](http://surgereviewojtjrlbofrfvyfp.surge.sh)

File: src/components/Notification/Notification.examples.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+<div>
+  <Notification
+    notifications={state.notifications}
+    selectedNotification={state.selectedNotification}
+    onNotificationClicked={notification => {
+        if (notification) {
+            console.log('clicked: '+notification)
+            console.log('READ: ' + notification.isMsgRead)
+            let notificationIdx = undefined
+            for (let i = 0; i < notifications.length; i++) {
```
Comment (ggascoigne) [inline src/components/Notification/Notification.examples.md]: why aren't you just using `notifications.find()`?

File: src/components/Notification/Notification.examples.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  notifications: notifications
+};
+
+<div>
+  <Notification
+    notifications={state.notifications}
+    selectedNotification={state.selectedNotification}
+    onNotificationClicked={notification => {
+        if (notification) {
+            console.log('clicked: '+notification)
```
Comment (ggascoigne) [inline src/components/Notification/Notification.examples.md]: delete tracing

File: src/styles/components/notification.css [hunk 3]
```diff
@@ -0,0 +1,223 @@
+@import '../variables.css';
+
+.notificationIcon{
```
Comment (ggascoigne) [inline src/styles/components/notification.css]: I thought that you were using BEM for you naming conventions - this doesn't seem to match that style at all.

File: src/components/Notification/Notification.examples.md (hunk 2 above)
Comment (andrewleyva) [reply src/components/Notification/Notification.examples.md]: I will before merging. This isn't done.

File: src/styles/components/notification.css (hunk 3 above)
Comment (andrewleyva) [reply src/styles/components/notification.css]: Yes, this clearly isn't BEM at all. I haven't had a chance to refactor Incedo's contribution.

**Comment by ggascoigne** [comment]
Woohoo! new surge deployment available for viewing! :tada: [surgereviewyznszulkrdcsxaq.surge.sh](http://surgereviewyznszulkrdcsxaq.surge.sh)

**Comment by andrewleyva** [comment]
@ggascoigne note that clear only works once per item in the example. There's probably a better way to handle data persistence in the example, but the component should be fine for an actual app.

**Pull request merged**